survey/models/survey.py

```diff
@@ -14,6 +16,11 @@ class Survey(models.Model):
     editable_answers = models.BooleanField(_("Users can edit their answers afterwards"), default=True)
     display_by_question = models.BooleanField(_("Display by question"))
     template = models.CharField(_("Template"), max_length=255, null=True, blank=True)
+    publish_date = models.DateField(_("Publicationdate"), blank=True, null=False, default=date.today())
+    default_survey_publishing_duration = settings.DEFAULT_SURVEY_PUBLISHING_DURATION
+    expire_date = models.DateField(
+        _("Expirationdate"), blank=True, null=False, default=date.today()+timedelta(
```

Comment:
We need a space in "Expirationdate" and "publicationdate" it's string that are visible by users.